gdnative-core/src/macros.rs

```diff
@@ -31,22 +31,13 @@ macro_rules! godot_gdnative_init {
         #[doc(hidden)]
         #[allow(unused_unsafe)]
         pub extern "C" fn $fn_name(options: *mut $crate::sys::godot_gdnative_init_options) {
-            unsafe {
-                let api = match $crate::GodotApi::from_raw((*options).api_struct) {
-                    Ok(api) => api,
-                    Err(e) => {
-                        $crate::report_init_error(options, e);
-                        return;
-                    }
-                };
-                $crate::GODOT_API = Some(api);
-                $crate::GDNATIVE_LIBRARY_SYS = Some((*options).gd_native_library);
+            let __init_success = unsafe { $crate::private::bind_api(options) };
+
+            if !__init_success {
```

Comment:
In the other macros that were changed in this PR this is "inlined" into the condition expression, is there a particular reason why this one here introduces a new variable bindings?

Reply:
This is just a nitpick, not a blocker for this PR, the rest looks all good! :+1:

Reply:
Not really. It's just an artifact of the transformation process. Inlined this one like the others.